Code/SimDivPickers/MaxMinPicker.h

```diff
@@ -66,7 +67,8 @@ class RDKIT_SIMDIVPICKERS_EXPORT MaxMinPicker : public DistPicker {
    *              poolSize*(poolSize-1)
    *   \param pickSize - the number items to pick from pool (<= poolSize)
    *   \param firstPicks - (optional)the first items in the pick list
-   *   \param seed - (optional) seed for the random number generator
+   *   \param seed - (optional) seed for the random number generator. 
+   *                 If this is <0 no seed will be used.
```

Comment:
Perhaps: 
```
a random seed will be used?
```


is more accurate.

Reply:
I thought about that wording but found it confusing.

Reply:
Perhaps "no deterministic seed will be used"

Reply:
Otherwise, feel free to merge.